code/modules/projectiles/pins.dm

```diff
@@ -241,9 +242,11 @@
 			return TRUE
 		if(H.get_bank_account())
 			credit_card_details = H.get_bank_account()
-		if(credit_card_details)
+		if(credit_card_details && !active_prompt)
 			var/license_request = input("Do you wish to pay [payment_amount] for license of this [gun.name]?", "License Purchase") as null|anything in list("Yes","No")
```

Comment:
You should probably remove the null from `as null|anything` to remove the cancel button, as it's basically just acting as a another No button anyway. Saves the `if(!license_request)` below too.

Reply:
just use alert instead wtf

Reply:
i didn't realize alert existed as an option before-hand, what does it actually do in comparison to input, or is it just a standardization thing?

Reply:
input supports actual text if you don't specific that list I think? Either works, but best to remove that cancel button either way.